broker/core/src/test/java/org/eclipse/kapua/broker/core/plugin/DefaultConnectorDescriptionProviderTest.java

```diff
@@ -12,32 +12,36 @@
  *******************************************************************************/
 package org.eclipse.kapua.broker.core.plugin;
 
-import org.eclipse.kapua.qa.markers.junit.JUnitTests;
-import org.hamcrest.core.IsInstanceOf;
+import org.eclipse.kapua.broker.core.setting.BrokerSetting;
+import org.eclipse.kapua.qa.markers.Categories;
+import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
-@Category(JUnitTests.class)
+@Category(Categories.junitTests.class)
 public class DefaultConnectorDescriptionProviderTest extends Assert {
 
     String[] connectorName;
 
+    @BeforeClass
+    public static void resettingBroker() {
+        BrokerSetting.resetInstance();
+    }
+
     @Before
     public void initialize() {
         connectorName = new String[]{null, "", "connector name", "name1234567890", "connector!@#$%^&*()_<>/"};
     }
 
-    @Test
-    public void getDescriptorTest() {
-        System.setProperty("broker.connector.descriptor.default.disable", "false");
-        System.setProperty("broker.connector.descriptor.configuration.uri", "file:src/test/resources/conector.descriptor/1.properties");
-        DefaultConnectorDescriptionProvider defaultConnectorDescriptionProvider = new DefaultConnectorDescriptionProvider();
-
-        for (String name : connectorName) {
-            assertThat("Instance of ConnectorDescriptor expected.", defaultConnectorDescriptionProvider.getDescriptor(name), IsInstanceOf.instanceOf(ConnectorDescriptor.class));
-        }
+    @After
+    public void cleanUp() {
+        System.clearProperty("broker.connector.descriptor.default.disable");
+        System.clearProperty("broker.connector.descriptor.configuration.uri");
+        System.out.println("broker-default-disable = " + System.getProperty("broker.connector.descriptor.default.disable"));
```

Comment:
Please use SLF4J logger or just remove them.

Same for all `System.out.println`